opencti-platform/opencti-front/src/private/components/settings/users/UserEditionOverview.tsx

```diff
@@ -132,16 +133,32 @@ UserEditionOverviewComponentProps
     });
   };
 
-  const handleSubmitField = (name: string, value: string | Date) => {
+  const handleSubmitField = (name: string, value: string) => {
     userValidation(t, userIsOnlyOrganizationAdmin)
       .validateAt(name, { [name]: value })
       .then(() => {
-        commitFieldPatch({
-          variables: {
-            id: user.id,
-            input: { key: name, value: value || '' },
-          },
-        });
+        // specific case for user confidence level that must be updated as a full object (we cannot field patch non-multiple objects for now)
+        // We pass the existing overrides for this user so they are unchanged
+        if (name === 'max_confidence') {
+          commitFieldPatch({
+            variables: {
+              id: user.id,
+              input: {
+                key: 'user_confidence_level',
+                object_path: '/user_confidence_level/max_confidence',
+                value: parseInt(value, 10),
+              },
+            },
```

Comment:
This is how we can now field patch the object properly.

FYI it also works with overrides with payload like:

```json
{
  "id": "26a02498-18e4-4b51-af18-44e5d8d6d5d9",
  "input": {
  "key": "user_confidence_level",
    "object_path": "/user_confidence_level/overrides/0/max_confidence",
    "value": 7
}
```